recipes/itmlogic/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "itmlogic" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fa65eb9bff2a0ff3798bc10d6962874aa594d4ed9826d4a36c23cb3b81115e51
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```

Comment:
Minor comment: if you use [grayskull](https://github.com/conda-incubator/grayskull) to create the recipes you will get the updated script line here. (And more nice things will be automatic populated for you.)